Kickstarter-iOS/Features/DashboardDeprecationBanner/DashboardDeprecationView.swift

```diff
@@ -0,0 +1,50 @@
+import Library
+import SwiftUI
+
+struct DashboardDeprecationView: View {
+  private let contentPadding = 12.0
+  private let imageSizeMultiplier = 1.5
+  private let deprecationDateText = formatted(dateString: "2023-08-14")
+
+  var body: some View {
+    HStack {
```

Comment:
Didn't see a need to use `(spacing: 0)` here. Double checked by running app a couple times.